wordpress-mobile/WordPress-iOS
# Remove target check from Podfile

Last year, I [added a section of code](https://github.com/wordpress-mobile/WordPress-iOS/pull/15202) to the WPiOS Podfile to silence the `IPHONEOS_DEPLOYMENT_TARGET` warnings that were being output by a lot of our pods:

```
The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99.
```

Recently, a tweak was made to the Podfile to add support for the Jetpack target. These changes included a [check](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/Podfile#L477) to the previous deployment target fix:

```
if target.name == "WordPress" || target.name == "Jetpack"
```

However, I noticed recently that many of the deployment target warnings had returned. I think that this is due to the above check, as the warning is no longer silenced for the extension and widget targets in the app (or anything other than the WordPress and Jetpack targets themselves). I think it is safe to remove this check – after taking it out, I no longer see the warnings and I am still able to build the WordPress and Jetpack targets successfully.

## To test

- Run `rake dependencies` and ensure there is no unexpected output
- Build the WordPress and Jetpack targets and ensure that a) they build and b) you don't see any `IPHONEOS_DEPLOYMENT_TARGET` warnings.

## Regression Notes

1. Potential unintended areas of impact

I don't think there are any. I think any issues would be surfaced during a `rake dependencies` or as build time warnings, but everything looks good to me.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

N/A

3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
